addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -244,7 +244,8 @@ patch(Chatter.prototype, {
 
     get childSubEnv() {
         const res = Object.assign(super.childSubEnv, { messageHighlight: this.messageHighlight });
-        res.inChatter.aside = this.props.isChatterAside;
+        assignGetter(res.inChatter, { aside: () => this.props.isChatterAside });
+        Object.assign(res.inChatter, { toggleComposer: this.toggleComposer.bind(this) });
```

Comment:
I made a utils function `assignGetter` so the business code stays easy to read and the subtlety is managed by the function.

This pattern to store contextual stuffs in and object of env and deep children accessing it is very useful and we are likely to use it more than once.